traefik/tests/pod-config_test.yaml

```diff
@@ -219,6 +219,25 @@ tests:
           path: spec.template.spec.containers[0].args
           content:
             "--providers.kubernetesgateway"
```

Comment:
Can you please remove the deprecated key `failure-domain.beta.kubernetes.io/zone` and replace it to `topology.kubernetes.io/zone`?

Reply:
Sorry, fogot about this one. Changed